x-pack/plugins/ingest_manager/server/services/epm/packages/index.ts

```diff
@@ -26,6 +26,16 @@ export {
 export { installKibanaAssets, installPackage, ensureInstalledPackage } from './install';
 export { removeInstallation } from './remove';
 
+type RequiredPackage = 'system' | 'endpoint';
+const requiredPackages: Record<RequiredPackage, boolean> = {
+  system: true,
+  endpoint: true,
+};
+
+export function isRequiredPackage(value: string): value is RequiredPackage {
```

Comment:
This tests based only on the package name, not name + version. We can add versions in later if we need to

Reply:
++